bedrock/foundation/templates/foundation/annualreport/2016/index.html

```diff
@@ -314,6 +318,8 @@ <h2>{{ _('Mozilla Corporation and Mozilla Foundation: Maintaining Sustainability
             {% endtrans %}
             </p>
 
+            {{ high_res_img('foundation/annualreport/2016/2016-report.png', {'alt': _('Consolidating Statement of Activities and Change in Net Assets'), 'width': '570', 'height': '389'}) }}
```

Comment:
The title seems like inadequate alt text. It doesn't really replace the image. Short of building this as a properly accessible table, I'd probably opt for empty alt.